q2_fmt/plugin_setup.py

```diff
@@ -360,4 +368,62 @@
     }
 )
 
+plugin.methods.register_function(
+    function=q2_fmt.peds_simulation,
+    inputs={'table': FeatureTable[Frequency | RelativeFrequency |
+                                  PresenceAbsence]},
+    parameters={'metadata': Metadata, 'time_column': Str,
+                'reference_column': Str, 'subject_column': T_subject,
```

Comment:
Just for readability:

```suggestion
                'reference_column': Str, 
                'subject_column': T_subject,
```